modules/apps/frontend-theme-locos/locos-theme/src/css/_banner.scss

```diff
@@ -0,0 +1,93 @@
+%astyle {
+  color: #999990;
+  font-family: 'Poppins', sans-serif;
+}
+%ahover {
+  text-decoration: none;
+  color: #f28959;
+  border-bottom: 3px solid #f28959;
+  -webkit-transition:border-bottom .5s ease;
+  -moz-transition:border-bottom .5s ease;
+  -ms-transition:border-bottom .5s ease;
+  -o-transition:border-bottom .5s ease;
+  transition:border-bottom .5s ease;
+}
+
+#sign-in {
+  order: 3;
+  font-size: 20px;
+  margin: 20px 20px 25px 20px;
+  @extend %astyle;
```

Comment:
Missing line break between CSS blocks.

Reply:
do you mean between order and font-size?  Or margin and @extend?

Reply:
After ```@extend``` and ```&:hover```.